ontologies/vocabs.ttl

```diff
@@ -350,8 +343,10 @@ Specifies a Windows Domain account"""^^xsd:string ;
 
 gist:_AccountType_windows_local
 	a gist:AccountType ;
-	rdfs:label "Windows Local"^^xsd:string ;
-	skos:altLabel "Windows-Local"^^xsd:string ;
+	skos:altLabel
+		"Windows Local"^^xsd:string ,
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?